benchmark/plotting/metrics.py

```diff
@@ -163,6 +163,11 @@ def mean_latency(attrs):
         "description": "Mean latency across queries",
         "function": lambda true_nn, run_nn, metrics, run_attrs: mean_latency(run_attrs),  
         "worst": float("inf")
-    }
+    },
+    "search_times": {
+        "description": "List of consecutive search times for the same run parameter",
+        "function": lambda true_nn, run_nn, metrics, run_attrs: run_attrs["search_times"], 
```

Comment:
For backwards compatability, this could be `run_attrs.get("search_times", "NA")` (or whatever should be the default string)